 core: ignore recovery resource zip
Won't be shipping recovery or anything anyways

diff --git a/core/Makefile b/core/Makefile
index 5403774..0e60013 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1168,24 +1168,7 @@ else
 recovery_wipe :=
 endif
 
-# Prior to A/B update, we used to have:
-#   boot.img + recovery-from-boot.p + recovery-resource.dat = recovery.img.
-# recovery-resource.dat is needed only if we carry a patch of the boot and
-# recovery images and invoke install-recovery.sh on the first boot post an
-# OTA update.
-#
-# We no longer need that if one of the following conditions holds:
-#   a) We carry a full copy of the recovery image
-#      (BOARD_USES_FULL_RECOVERY_IMAGE = true);
-#   b) We build a single image that contains boot and recovery both
-#      (BOARD_USES_RECOVERY_AS_BOOT = true).
-
-ifeq (,$(filter true, $(BOARD_USES_FULL_RECOVERY_IMAGE) $(BOARD_USES_RECOVERY_AS_BOOT) $(TARGET_NOT_USE_GZIP_RECOVERY_RAMDISK)))
-# Named '.dat' so we don't attempt to use imgdiff for patching it.
-RECOVERY_RESOURCE_ZIP := $(TARGET_OUT)/etc/recovery-resource.dat
-else
 RECOVERY_RESOURCE_ZIP :=
-endif
 
 ifeq ($(recovery_fstab),)
   $(info No recovery.fstab for TARGET_DEVICE $(TARGET_DEVICE))
